src/platform/K32W/Logging.cpp

```diff
@@ -21,60 +24,56 @@
 static bool isLogInitialized;
 extern uint8_t gOtLogUartInstance;
 extern "C" void K32WWriteBlocking(const uint8_t * aBuf, uint32_t len);
+extern "C" uint32_t otPlatAlarmMilliGetNow(void);
 
 namespace chip {
 namespace Logging {
 namespace Platform {
 
-void GetMessageString(char * buf, uint8_t chipCategory, uint8_t otLevelLog)
+/* bufLen must have at least 4 (miliseconds timestamp) + 1 (category) +  */
```

Comment:
This "4" in the commment does not match the code?
```suggestion
/* bufLen must have at least TIMESTAMP_MAX_LEN_BYTES (milliseconds timestamp) + CATEGORY_MAX_LEN_BYTES (category) +  */
```

Reply:
Right, fixed the comment to match the code. See https://github.com/project-chip/connectedhomeip/pull/8451